namui-cli/electron/package.json

```diff
@@ -6,13 +6,15 @@
     "start": "electron .",
     "start:windows": "dotenv -e .wsl-env cross-var %WINDOWS_ELECTRON_EXE_PATH% .",
     "package": "electron-forge package",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "playwright test"
```

Comment:
For now, `npm test` is not used in namui's test. user directly call `npm test`.
We should decide where electron test be called

Reply:
electron test must be part of namui-cli test if namui-cli has control and management of namui runtime.

Reply:
Now, it will be tested on namui-cli test

Reply:
Test linked